proxy/test/utils.ts

```diff
@@ -0,0 +1,76 @@
+import type http from 'node:http'
+import { after, before } from 'node:test'
+
+interface TestServer {
+  address: string
+  baseUrl: string
+  port: number
+  server: http.Server
+}
+
+const __testServers: {
```

Comment:
Why not just `testServers`?

Reply:
I usually like to indicate that this variable is private, but let's be simple here! Changed back to `testServers`